Client/Frontend/Browser/TabManager.swift

```diff
@@ -356,17 +365,14 @@ class TabManager: NSObject {
     }
     
     private func saveTabOrder() {
-        let context = DataController.newBackgroundContext()
-        context.perform {
+        DataController.performBackgroundTask { context in
             for (i, tab) in self.allTabs.enumerated() {
-                guard let managedObject = TabMO.get(fromId: tab.id, context: context) else { 
+                guard let managedObject = TabMO.get(fromId: tab.id, context: context) else {
                     log.error("Error: Tab missing managed object")
                     continue
                 }
                 managedObject.order = Int16(i)
             }
-            
-            DataController.save(context: context)
```

Comment:
Why get rid of this save? `performBackgroundTask` doesn't auto-save, does it?

Reply:
It saves by default, I'm even considering removing `DataController.save()` function from code.
We almost always save after doing operations within a context, so this is a good default behavior to have imo